app/src/main/java/org/systers/mentorship/view/activities/SettingsActivity.kt

```diff
@@ -20,9 +26,27 @@ class SettingsActivity : BaseActivity() {
         supportActionBar?.setDisplayShowHomeEnabled(true);
 
         tvLogout.setOnClickListener {
-            preferenceManager.clear()
-            startActivity(Intent(this, LoginActivity::class.java))
-            finishAffinity()
+
+            val builder = AlertDialog.Builder(this)
+            builder.setTitle("CONFIRM LOGOUT")
```

Comment:
I think you should add **CONFIRM LOGOUT** text in **strings.xml** file. cc @isabelcosta